src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -122,6 +122,9 @@ declare -a args=(
     "target_cflags=$(format_gn_list "${target_cflags[@]}")"
     "target_cflags_c=$(format_gn_list "${target_cflags_c[@]}")"
     "target_cflags_cc=$(format_gn_list "${target_cflags_cc[@]}")"
+    'chip_project_config_include="<CHIPProjectConfig.h>"'
```

Comment:
Rather than hard-coding more and more settings in the shell script, I think we should add an `args.gni` file for the framework that contains those settings that aren't derived by the script itself from xcodebuild environment variables.

Reply:
I have moved most those additional lines into `config/darwin/args.gni`
About moving the others, that looks like a different issue to me.